special/OreDictEntryManager.php

```diff
@@ -123,13 +123,15 @@ private function createEntry(FormOptions $opts){
 	private function updateEntry(FormOptions $opts){
 		$dbw = wfGetDB(DB_MASTER);
 		$stuff = $dbw->select('ext_oredict_items', '*', array('entry_id' => $opts->getValue('entry_id')));
-		$result = $dbw->update('ext_oredict_items', array(
+		$ary = array(
 			//'tag_name' => $opts->getValue('tag_name'),
 			'item_name' => $opts->getValue('item_name'),
 			'mod_name' => $opts->getValue('mod_name'),
 			'grid_params' => $opts->getValue('grid_params'),
 			'flags' => $opts->getValue('flags')
-		), array('entry_id' => $opts->getValue('entry_id')));
+		);
+		$tableName = $dbw->tableName('ext_oredict_items');
```

Comment:
This won't matter on Hydra, but if other sites use this it will cause a double prefixed table name down below on $dbw->delete().